src/modality-specific-files/task-events.md

```diff
@@ -309,3 +309,97 @@ in the accompanying JSON sidecar as follows (based on the example of the previou
     "VisionCorrection": "lenses"
 }
 ```
+
+### Continuously-sampled, stimulus-related signals
+
+!!! example "Example datasets"
+
+      At the time of writing, there are no [example datasets][bids-examples] showcasing
+      stimulus-related signals.
+      However, the following datasets shared on [OpenNeuro](https://openneuro.org)
+      include these type of signals and may be used (with caution, because they are not
+      "official" examples) as a reference when curating a new dataset:
+
+      -   [*Trial timing for multivariate pattern analysis*][ds000238]
```

Comment:
Thanks for looking this up: I can try to open a PR to the bids example by adapting this dataset to make sure we got an official example.

Reply:
That'd be lovely

Reply:
There is remark failure for the markdown formatting: seems that the admonition are also messing up putting "markdown links" at the bottom of the page when using an admonition.

Not 100% sure about this though.

Reply:
OK there is technically a bids example with stim files: the synthetic example.

https://github.com/bids-standard/bids-examples/blob/master/synthetic/sub-01/ses-01/func/sub-01_ses-01_task-nback_run-01_stim.tsv.gz

I still think it'd be nice for users to have an example that links also to a real dataset.

Thoughts? @effigies @oesteban @sappelhoff

Reply:
Sure.